packages/controller/src/lib/session.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-
+/** @type any */
```

Comment:
Can I help give this a better type?

Reply:
The type actually wasn't important for me because I did not want to port this file to ts now. Without the type annotation ts is not able to build the file.

Reply:
ok so fix later then